Gateway/Request/RefundRequest.php

```diff
@@ -73,4 +70,16 @@ public function build(array $buildSubject)
             'xRefNum' => $payment->getParentTransactionId(),
         ];
     }
+
+    /**
+     * Format price to 0.00 format
+     *
+     * @param mixed $price
+     * @return string
+     * @since 100.1.0
+     */
+    public function formatPrice($price)
```

Comment:
Would it make sense to have this in a central location instead of duplicating it? Is that at all possible?

Reply:
ohh yes, It is possible, I will update it

Reply:
I updated code. Please check this commit [62a9a7](https://github.com/cardknox/magento2_cardknox/pull/27/commits/62a9a793ea49098cfd180c3dcc01cc4185ec7814)